fabric-content-registries-v1/src/main/java/net/fabricmc/fabric/api/content/registry/v1/FlammableBlockRegistry.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.content.registry.v1;
+
+import net.minecraft.block.Block;
+import net.minecraft.tag.Tag;
+
+import net.fabricmc.fabric.api.content.registry.v1.util.Taggable2ObjectMap;
+import net.fabricmc.fabric.impl.content.registry.FlammableBlockRegistryImpl;
+
+/**
+ * Registry of Blocks that can be set on fire.
+ */
+public interface FlammableBlockRegistry extends Taggable2ObjectMap<Block, FlammableBlockRegistry.Entry> {
+	/**
+	 * Registry of Blocks that can be set on fire by {@link net.minecraft.block.Blocks#FIRE}.
+	 */
+	FlammableBlockRegistry FIRE_INSTANCE = FlammableBlockRegistryImpl.FIRE_INSTANCE;
+
+	default void add(Block block, int burn, int spread) {
+		this.add(block, new Entry(burn, spread));
+	}
+
+	default void add(Tag<Block> tag, int burn, int spread) {
+		this.add(tag, new Entry(burn, spread));
+	}
+
+	final class Entry {
```

Comment:
this should be in the impl if its not exposed

Reply:
It can be got using the `get` method and the values extracted from, so it's api

Reply:
Can we call this `Flammability` than some generic `Entry`?

Reply:
It's FlammableBlockRegistry.Entry